README.md

```diff
@@ -4,29 +4,6 @@ Now, when you felt the power of the Powershell, you know that you can do basical
 
 In this task, you will implement a Powershell script, which deploys an Azure Virtual Machine to your subscription. 
 
-## How to complete tasks in this module 
-
-Tasks in this module are relying on 2 PowerShell scripts: 
-
-- `scripts/generate-artifacts.ps1` generates the task "artifacts" and uploads them to cloud storage. An "artifact" is evidence of a task completed by you. Each task will have its own script, which will gather the required artifacts. The script also adds a link to the generated artifact in the `artifacts.json` file in this repository — make sure to commit changes to this file after you run the script. 
-- `scripts/validate-artifacts.ps1` validates the artifacts generated by the first script. It loads information about the task artifacts from the `artifacts.json` file.
-
-Here is how to complete tasks in this module:
-
-1. Clone task repository
-
-2. Make sure you completed steps, described in the Prerequisites section
-
-3. Complete the task, described in the Requirements section 
-
-4. Run `scripts/generate-artifacts.ps1` to generate task artifacts. Script will update the file `artifacts.json` in this repo. 
-
-5. Run `scripts/validate-artifacts.ps1` to test yourself. If tests are failing - follow the recomendation from the test script error message to fix or re-deploy your infrastructure. When you will be ready to test yourself again - **re-generate the artifacts** (step 4) and re-run tests again. 
-
-6. When all tests will pass - commit your changes and submit the solution for a review. 
-
-Pro tip: if you stuck with any of the implementation steps - run `scripts/generate-artifacts.ps1` and `scripts/validate-artifacts.ps1`. The validation script might give you a hint on what you should do.  
-
 ## Prerequisites
```

Comment:
3. In the `mate-resources` resource group, create a storage account (any name) and a `task-artifacts` container.

Reply:
PowerShell everywhere (not Powershell)